terrain/voxel_terrain.cpp

```diff
@@ -769,6 +771,7 @@ void VoxelTerrain::_process() {
 				out_of_range_box.for_each_cell([=](Vector3i bpos) {
 					// Unload block
 					immerge_block(bpos);
+					emit_signal(SIGNAL_BLOCK_UNLOADED, bpos.to_vec3(), 0); // TODO: only lod0 here, or need to grab from block?
```

Comment:
I'll update to use `VoxelStringNames`, and put emit inside `immerge_block`.
My preference is also to not send LOD at all in VoxelTerrain, just wasn't sure if needed or not. I'll remove LOD from `VoxelTerrain` signal when implementing `VoxelLODTerrain` signals.

My current use case is pathfinding over the "ground", so I think I don't care about blocks that are only air/dirt/other flood fill. What were you thinking?

Reply:
It would be possible to avoid emitting if the block is uniform, though you can check that in the function for now. Did you test your use case with that functionality to see how it goes?

Reply:
As far as I can tell, we can't check if a block is uniform from script, because we don't expose the functionality of getting block from terrain anywhere. `VoxelTool` only has `get_voxel`, no `get_block`. 

I did add my own `get_block` to `VoxelTool` however. There is no time difference between having `is_uniform` in `VoxelTerrain` before sending signal vs just check `is_uniform` via script, so I think we don't need to have check be module.